datasets/bgt/begroeideterreindelen/v1.0.0.json

```diff
@@ -0,0 +1,115 @@
+{
+  "id": "begroeideterreindelen",
+  "title":"begroeide terreindelen",
+  "description": "Kleinste functioneel onafhankelijk stukje van een terrein, dat er binnen het objecttype Terrein van NEN 3610 wordt onderscheiden, met aaneengesloten vegetatie. - Groep: terrein - Code: 140",
+  "type": "table",
+  "version": "1.0.0",
+  "crs": "EPSG:28992",
+  "auth": "OPENBAAR",
+  "temporal": {
+    "identifier": "volgnummer",
+    "dimensions": {
+      "geldigOp": [
+        "objectBegintijd",
+        "objectEindtijd"
+      ]
+    }
+  },
+  "schema": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "additionalProperties": false,
+    "mainGeometry": "geometrie2D",
+    "identifier": [
+      "identificatie",
+      "volgnummer"
+    ],
+    "required": [
+      "schema",
+      "identificatie"
+    ],
+    "display": "identificatie",
+    "properties": {
+      "schema": {
+        "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+      },
+      "bgtFunctie": {
+        "type": "string",
+        "description": "Specificatie van de bgt functie van ondersteunend wegdeel."
+      },
+      "bgtFysiekVoorkomen": {
+        "type": "string",
+        "description": "Classificatie van het vegetatiedek, ingedeeld naar soort vegetatie."
+      },
+      "plusFysiekVoorkomen": {
+        "type": "string",
+        "description": "Nadere classificatie van het vegetatiedek, ingedeeld naar soort vegetatie."
+      },
+      "opTalud": {
+        "type": "string",
```

Comment:
Is dit geen boolean?

Reply:
nee, want ik weet niet gegarandeerd zeker dat er geen NULL in zit